src/bgen/BindingTouch.cs

```diff
@@ -666,6 +649,21 @@ void Compile (List<string> arguments, int errorCode, string tmpdir)
 			Console.WriteLine (output);
 	}
 
+	Assembly? TryLoadApi (string? name)
```

Comment:
The typical pattern for Try\* APIs is a bool return value + out parameter for the actual value:

```cs
bool TryLoadApi (string? name, [NotNullWhen (true)] out Assembly? assembly)
{
    assembly = null;
    if (string.IsNullOrEmpty (name))
        return false;
    // ...
}
```